### PR TITLE
Improve Documentation for Custom Starters in kedro new (#2618)

### PR DESCRIPTION
**Pull request by doxenix**
Subject:

This pull request aims to enhance the user experience and clarity of documentation within kedro new command for custom starters. Specifically, it addresses issue #2618

### CODE REVIEW

File: docs/source/starters/starters.md [hunk 1]
```diff
@@ -79,5 +85,11 @@ kedro new --config=my_kedro_project.yml --starter=spaceflights-pandas
 
 This option is useful when the starter requires more configuration than the default mode requires.
 
-## Create a starter
+## Custom starter
```
Comment (astrojuanlu) [inline docs/source/starters/starters.md:88]:
```suggestion
## Create a custom starter
```

**Review by astrojuanlu — COMMENTED** [review]
Thanks a lot for this PR @doxenix ! Left a couple of minor comments

File: docs/source/starters/starters.md (hunk 1 above)
Comment (doxenix) [reply docs/source/starters/starters.md]: changed

File: docs/source/starters/starters.md [hunk 2]
```diff
@@ -79,5 +85,11 @@ kedro new --config=my_kedro_project.yml --starter=spaceflights-pandas
...
 You can build your own starters for reuse within a project or team, as described in the [how to create a Kedro starter](../starters/create_a_starter.md) documentation.
+
+To create project with created custom starter template, run:
```
[the remark this answers is not in the document]
Comment (doxenix) [reply docs/source/starters/starters.md]: changed

**Review by astrojuanlu — APPROVED** [review]
Thank you @doxenix ! Feel free to add your name under https://github.com/doxenix/kedro/blob/fix/2618/RELEASE.md#community-contributions

**Review by merelcht — APPROVED** [review]
Thanks a lot for this contribution @doxenix ⭐

**Pull request merged**